taxi_service/settings.py

```diff
@@ -0,0 +1,126 @@
+"""
+Django settings for taxi_service project.
+
+Generated by 'django-admin startproject' using Django 5.1.4.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.1/ref/settings/
+"""
+
+from pathlib import Path
+
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
```

Comment:
The `DEBUG` setting is currently set to `True`. Remember to set this to `False` in production environments to avoid exposing sensitive information.